upup/models/cloudup/resources/addons/networking.weave/v1.9.0.yaml

```diff
@@ -41,19 +41,29 @@ spec:
             - name: weavedb
               mountPath: /weavedb
             - name: cni-bin
-              mountPath: /opt
+              mountPath: /host/opt
```

Comment:
I'm reasonably sure `/opt` is in the LSB.  In any case, docker will create it if not...